doc/whats_new.rst

```diff
@@ -0,0 +1,47 @@
+:orphan:
+
+.. include:: links.inc
+
+.. _whats_new:
+
+.. currentmodule:: spykes
+
+.. _current:
+
+Current
+-------
+
+Changelog
+~~~~~~~~~
+
+    - ...
+
+BUG
+~~~
+
+    - ...
+
+API
+~~~
+
+    - ...
+
+
+Version 0.1
+-----------
+
+Changelog
+~~~~~~~~~
+
+    - Add NeuroVis class by `Hugo Fernandes`_ and `Pavan Ramkumar`_
+    - Add NeuroPop class by `Hugo Fernandes`_ and `Pavan Ramkumar`_
+    - Add PopVis class by `Mayank Agrawal`_
+    - Add STRF class by `Pavan Ramkumar`_
+    - Add Sphinx Documentation `Mayank Agrawal`_
+    - Add Travis CI by `Mayank Agrawal`_
+    - Add Circle CI by `Mainak Jas`_ and `Mayank Agrawal`_
```

Comment:
I wouldn't put this in the changelog :) Just new features etc. Users don't care about Circle CI